src/app/components/client/Confetti.module.scss

```diff
@@ -1,9 +1,14 @@
 .confetti {
   background: transparent;
+  display: none;
   height: 100%;
   position: absolute;
   width: 100%;
   pointer-events: none;
   user-select: none;
   z-index: 1;
+
+  @media (prefers-reduced-motion: no-preference) {
+    display: none;
```

Comment:
This probably shouldn't be `none`? I'm not seeing any confetti at all now :)

Reply:
Though it might be nice if it’s a secret surprise party. Thinking about it again, let’s invite more people: [671d129](https://github.com/mozilla/blurts-server/pull/3753/commits/671d129c8ac39664a61aae2bf32daf4846de9f5a).